{"name": "登录失败场景", "status": "passed", "description": "登录失败场景", "attachments": [{"name": "stdout", "source": "6a3747df-cbc2-4437-943d-90f3adb78bbb-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "browser_name", "value": "'chromium'"}, {"name": "username", "value": "'yoyox1x2x3'"}, {"name": "password", "value": "'12345678'"}, {"name": "title", "value": "'输入错误的账号'"}], "start": 1713513227804, "stop": 1713513228415, "uuid": "f3dffbc3-55b3-4c84-85c6-f9e054ad2bde", "historyId": "42f0748272a7e91bead0e2c3f5b1e6e5", "testCaseId": "0171dfb4726b6d084b4a08882eabf7ca", "fullName": "cases.test_login.TestLogin#test_login_error", "labels": [{"name": "feature", "value": "登录功能"}, {"name": "parentSuite", "value": "cases"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-HJ487C8"}, {"name": "thread", "value": "1228-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "cases.test_login"}]}